lib/jss.map

```diff
@@ -485,3 +485,10 @@ Java_org_mozilla_jss_nss_SECErrors_getUntrustedCert;
     local:
         *;
 };
+JSS_4.7.1 {
+    global:
+Java_org_mozilla_jss_pkcs11_PK11Cert_duplicate;
+Java_org_mozilla_jss_pkcs11_PK11PrivKey_duplicate;
```

Comment:
It's not really an issue, but usually this type of operation is implemented either as `clone()` or copy constructor. Do we want to use that instead of `duplicate()`? I'll let you decide.